app/src/main/java/org/wikipedia/views/FaceAndColorDetectImageView.kt

```diff
@@ -55,18 +55,22 @@ class FaceAndColorDetectImageView constructor(context: Context, attrs: Attribute
 
                 override fun onResourceReady(resource: Drawable?, model: Any, target: Target<Drawable?>, dataSource: DataSource, isFirstResource: Boolean): Boolean {
                     if (resource is BitmapDrawable && resource.bitmap != null) {
-                        listener.onImageLoaded(Palette.from(resource.bitmap).generate())
+                        listener.onImageLoaded(Palette.from(resource.bitmap).generate(), resource.bitmap.width, resource.bitmap.height)
                     } else {
                         listener.onImageFailed()
                     }
                     return false
                 }
             })
         }
-        builder = if (shouldDetectFace(uri)) {
-            builder.transform(if (roundedCorners) FACE_DETECT_TRANSFORM_AND_ROUNDED_CORNERS else FACE_DETECT_TRANSFORM)
+        if (scaleType == ScaleType.FIT_CENTER) {
+            builder = builder.transform(WhiteBackgroundTransformation())
```

Comment:
Looks like this will make some of the article leading images show as CENTER_INSIDE with white background. Is this expected? (e.g. DMX(rapper) article).